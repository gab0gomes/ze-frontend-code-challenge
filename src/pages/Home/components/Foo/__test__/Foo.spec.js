import { mount, createLocalVue } from '@vue/test-utils'

import Foo from '../index'

const localVue = createLocalVue()

describe('Foo', () => {
  it('render correctly', async () => {
    const wrapper = mount(Foo, { localVue })

    expect(wrapper.element).toMatchSnapshot()

    wrapper.destroy()
  })
})
