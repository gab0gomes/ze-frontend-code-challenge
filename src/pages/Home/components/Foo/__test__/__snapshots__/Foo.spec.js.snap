// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo render correctly 1`] = `
<div>
  Foo
</div>
`;
